scripts/verify-sdk-builds.sh

```
@@ -0,0 +1,53 @@
+echo "=== Verifying Go SDK builds ==="
+# go mod tidy updates go.sum for any deps introduced by custom files (e.g. ajson)
+# that Speakeasy didn't see during generation.
+(cd api/go && go mod tidy && go build ./...)
+echo "Go SDK: OK"
+
+echo "=== Verifying TypeScript SDK builds ==="
+(cd api/typescript && npm install --ignore-scripts && npm run build)
+echo "TypeScript SDK: OK"
+
+echo "=== Verifying Python SDK syntax ==="
+find api/python/src -name "*.py" | xargs python3 -m py_compile
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)